src/__tests__/reporter/reporter.spec.ts

```diff
@@ -0,0 +1,196 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+// @ts-ignore
+import RPClient from '@reportportal/client-javascript';
+import { Reporter } from '../../reporter';
+import { getDefaultMockConfig, RPClientMock } from './RPClientMock';
+import { StartLaunchRQ, StartTestItemRQ } from '../../models';
+import { STATUSES, TEST_ITEM_TYPES } from '../../constants';
+import { getStartLaunchObj } from '../../utils';
+
+const setupReporter = (fields: Array<string> = []): Reporter => {
+  const config = getDefaultMockConfig();
```

Comment:
You use the same function as on line 43. It will be better to move `const config = getDefaultMockConfig ();` before `setupReporter` and use it inside of it.